### PR TITLE
[#160165192] Pin screen status bar color and background

### PR DESCRIPTION
**Pull request by matteodesanti**
Ok for now..

TODO: 
- find a generic way to catch the screen background color and adjust the status bar backgorund and color

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Remember to fix the PR title by removing WIP wording when ready
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#160165192](https://www.pivotaltracker.com/story/show/160165192): Come CIT vorrei vedere la status bar del mio dispositivo con un colore coerente con quello della header dell'app

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/PinLoginScreen.tsx [hunk 1]
```diff
@@ -117,6 +120,10 @@ class PinLoginScreen extends React.Component<Props> {
     };
     return (
       <BaseScreenComponent primary={true} contextualHelp={contextualHelp}>
+        <StatusBar
```
Comment (cloudify) [inline ts/screens/PinLoginScreen.tsx]: this adds the status bar to all screens based on `BaseScreenComponent` (almost all), is that what we want?

Comment (fpersico) [reply ts/screens/PinLoginScreen.tsx]: @cloudify He is not adding something to the BaseScreenComponent file, we are in the PinLoginScreen. Also the StatusBar component is not really a "rendered" component, you can put 10 StatusBar in each screen and only one is rendered, the last one dictate the android/ios bar background/text color.

Comment (cloudify) [reply ts/screens/PinLoginScreen.tsx]: sorry you are right :)

**Pull request merged**